ci/Conformance.md

```diff
@@ -0,0 +1,67 @@
+# Conformance results
```

Comment:
Provide a summary of the steps needed and the purpose of the conformance test.

Creating a flow diagram for extra credit.